notebooks/WET/WET-004/WET-004-ComCam.ipynb

```diff
@@ -0,0 +1,385 @@
+{
```

Comment:
<p>It would  help to have here more explanation - are you comparing Zk estimation for these two blends against the average of that CCD?  Or average of nine CCDs?  Looking at the image I'm not sure.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>